livekit-ffi/src/cabi.rs

```diff
@@ -15,13 +15,23 @@ pub type FfiCallbackFn = unsafe extern "C" fn(*const u8, usize);
 ///
 /// The foreign language must only provide valid pointers
 #[no_mangle]
-pub unsafe extern "C" fn livekit_ffi_initialize(cb: FfiCallbackFn, capture_logs: bool) {
+pub unsafe extern "C" fn livekit_ffi_initialize(
+    cb: FfiCallbackFn,
+    capture_logs: bool,
+    sdk: &'static [u8; 16],
+    sdk_version: &'static [u8; 16],
```

Comment:
i got warnings when using str or String here that they aren't FFI-safe so I used `[u8; 16]` which meant i also needed to handle nulls below so I made sdk and sdk version optional. I'm going to go ahead and do the python and maybe the unity implementation before I merge this, since I believe this bit here is used in those but not node, so its not tested yet